### PR TITLE
Beacon config rework

### LINKED ISSUE #1: Switch APRS SSID to new values
Please switch source and location SSIDs to those listed in the [OreSat APRS Header](https://docs.google.com/document/d/1gYe_WxVkGzgNJl752F_rJQmdJALE3-15hB1A2Xo8JYc/edit#heading=h.d08m74ici087) document.

### PR DESCRIPTION
**Pull request by ryanpdx**
All beacon AX.25 header fields are now set in the `beacon.yaml` file. 

The beacon header table was move to each oresat mission beacon definition page as some of the values are different for different OreSat missions.

temp

closes #1

### CODE REVIEW

File: oresat_configs/_yaml_to_od.py [hunk 1]
```diff
@@ -670,6 +670,14 @@ def _gen_od_db(oresat_id: OreSatId, beacon_def: dict, configs: dict) -> dict:
         od["satellite_id"].default = oresat_id.value
         if node_id == NodeId.C3:
             od["beacon"]["revision"].default = beacon_def["revision"]
+            od["beacon"]["dest_callsign"].default = beacon_def["ax25"]["dest_callsign"]
+            od["beacon"]["dest_ssid"].default = beacon_def["ax25"]["dest_ssid"]
+            od["beacon"]["src_callsign"].default = beacon_def["ax25"]["src_callsign"]
+            od["beacon"]["src_ssid"].default = beacon_def["ax25"]["src_ssid"]
+            od["beacon"]["src_ssid"].default = beacon_def["ax25"]["src_ssid"]
```
Comment (aSmig) [inline oresat_configs/_yaml_to_od.py:677]: 676 and 677 duplicate line?

File: oresat_configs/oresat0/beacon.yaml [hunk 2]
```diff
@@ -1,4 +1,12 @@
 revision: 0
+ax25:
+  dest_callsign: SPACE
+  dest_ssid: 0
+  src_callsign: KJ7SAT
+  src_ssid: 0
+  control: 0x3 # ui-frame
+  pid: 0xf0 # no L3 protocol
+  command: true
```
Comment (aSmig) [inline oresat_configs/oresat0/beacon.yaml:9]: OreSat0 did not set C-Bits in src or dest SSID bytes.

File: docs/scripts/gen_beacon_rst.py [hunk 3]
```diff
@@ -23,30 +25,118 @@
...
 
+    c3_od = config.od_db[NodeId.C3]
+    src_callsign = c3_od["beacon"]["src_callsign"].value
+    src_callsign = src_callsign + " " * (6 - len(src_callsign))
+    src_ssid = c3_od["beacon"]["src_ssid"].value
+    dest_callsign = c3_od["beacon"]["dest_callsign"].value
+    dest_callsign = dest_callsign + " " * (6 - len(dest_callsign))
+    dest_ssid = c3_od["beacon"]["dest_ssid"].value
+    command = c3_od["beacon"]["command"].value
+    reversed_bits = 0b0110_0000
```
Comment (aSmig) [inline docs/scripts/gen_beacon_rst.py:58]: reversed == reserved?

File: docs/scripts/gen_beacon_rst.py [hunk 4]
```diff
@@ -23,30 +25,118 @@
...
+    control = c3_od["beacon"]["control"].value
+    pid = c3_od["beacon"]["pid"].value
+
+    dest_ssid = (dest_ssid << 1) | (int(not command) << 7) | reversed_bits
```
Comment (aSmig) [inline docs/scripts/gen_beacon_rst.py:62]: This is the part that makes representing OreSat0 hard where neither C-Bit is set. I'm not really sure what to suggest. Maybe check to see if command is defined and then only define it as true/false on missions that use a C-Bit?

File: docs/scripts/gen_beacon_rst.py [hunk 5]
```diff
@@ -23,30 +25,118 @@
...
+    src_ssid = (src_ssid << 1) | (int(command) << 7) | reversed_bits | 1
```
Comment (aSmig) [inline docs/scripts/gen_beacon_rst.py:63]: Command packets should have only the dest C-Bit set, so I think the not from the line above needs to move down here.

File: oresat_configs/oresat0_5/beacon.yaml [hunk 6]
```diff
@@ -1,4 +1,12 @@
 revision: 0
+ax25:
+  dest_callsign: SPACE
+  dest_ssid: 0 # primary station
+  src_callsign: KJ7SAT
+  src_ssid: 11 # balloons, aircraft, spacecraft, etc
+  control: 0x3 # ui-frame
+  pid: 0xf0 # no L3 protocol
+  command: true
```
Comment (aSmig) [inline oresat_configs/oresat0_5/beacon.yaml:9]: I suggest having this be a response packet instead of a command packet. Or `command: false`

**Review by aSmig — CHANGES_REQUESTED** [review]
Looking pretty good with a couple minor issues. Sorry for all the extra work over so few bits.

File: docs/scripts/gen_beacon_rst.py (hunk 4 above)
Comment (aSmig) [reply docs/scripts/gen_beacon_rst.py:62]: Hmm, what if there is a second boolean called `response` where `command` only sets the dest SSID C-Bit and `response` only sets the src SSID C-Bit? Then OreSat0 can be false for both, and OreSat0.5 can be true for `response`.

Comment (ryanpdx) [reply docs/scripts/gen_beacon_rst.py]: That sounds good, it is probably the cleanest way to do it

**Review by aSmig — APPROVED** [review]
Looks good!

**Pull request merged**